katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
I like 'normalised_name'. You could also call it 'slug', which is the term
used by many web content management systems  when turning a title into an
URL (and also used in the newspaper industry for a similar purpose, which
is where it came from)

Cheers
Neilen

On Wed, Jul 22, 2015 at 4:19 PM, Lize van den Heever <
notifications@github.com> wrote:

> In katcp/resource.py
> <https://github.com/ska-sa/katcp-python/pull/117#discussion_r35217268>:
>
> >          return self._name
> >
> >      @property
> > +    def python_id(self):
>
> Hmm - no specific preference. But may be "normalised_name" to make it less
> python specific. Or what about something very different like
> "sensor_handle" or "sensor_unih" for "unique handle". Or is it just not
> good to introduce such new terminology now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ska-sa/katcp-python/pull/117/files#r35217268>.
>